packages/removable-tags/src/components/RemovableTags.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import StyledTag from "./styledComponent/StyledTag";
+
+class RemovableTags extends Component {
+  state = {
+    isVisible: true
+  };
+  onCloseHandler = this.onCloseHandler.bind(this);
+
+  onCloseHandler() {
+    this.setState({ isVisible: false });
+    this.props.onClose();
+  }
+
+  render() {
+    if (!this.state.isVisible) {
+      return null;
+    }
```

Comment:
I don't know if this is necessary. The logic to hide the component should be handled by the container.

But let's check with the team what's their opinion before removing.

Reply:
Agreed.